libcloud/compute/drivers/ec2.py

```diff
@@ -5545,6 +5586,15 @@ class EC2EUNodeDriver(EC2NodeDriver):
     _region = 'eu-west-1'
 
 
+class EC2EUCentralNodeDriver(EC2NodeDriver):
```

Comment:
I just noticed an issue while testing this change - this code will only use signature v4 if you are using old and deprecated class based approach. It won't use it if you use the `region` argument.

```python
EC2 = get_driver(Provider.EC2)
driver = EC2('...', '...', region='eu-central-1')

```

```bash
 -------- begin 140518062231056 request ----------
curl -i -X GET -H 'Host: ec2.eu-central-1.amazonaws.com' -H 'X-LC-Request-ID: 140518062231056' -H 'Accept-Encoding: gzip,deflate' -H 'User-Agent: libcloud/0.17.1-dev (Amazon EC2) ' --compress 'https://ec2.eu-central-1.amazonaws.com:443/?SignatureVersion=2&AWS...'
```

I will work on a fix.